docs/items/items.md

```diff
@@ -30,4 +30,33 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+There are items Minecraft prefers to render by coloring one single texture. Leather helmets, spawn eggs, potions and other.
+
+### Item Color Handler
+
+Item color handler is just an instance of `IItemColor`. The interface contains only one method:
+
+```java
+getColorFromItemstack(
+  ItemStack stack, 
+  int tintIndex)
+```
+
+It provides ItemStack of item that is being painted and the tint index. 
+
+When coloring an ItemBlock, the tint index is the same thing as it is with blocks, but if an Item is being painted, the tint index value indicates which layer of an item is being painted. Layers of item are specified in its model JSON file and are layers of its texture.
```

Comment:
I'd reword this (and remove all the uses of "painted") to something along the lines of

```markdown
The tint index parameter has a different meaning depending on whether the item being colored is an `ItemBlock`.

For an `ItemBlock`, the tint index is exactly the same as it would be for its [block variant](../blocks/blocks.md#coloring-a-block).

For other kinds of items, the tint index indicates which texture layer is currently being drawn. These layers have the same meaning as they do in the item's model JSON file.
```